cognite/powerops/custom_modules/power_model_v1/data_models/containers/PriceProduction.container.yaml

```diff
@@ -0,0 +1,29 @@
+space: "{{powerops_models}}"
+externalId: PriceProduction
+name: PriceProduction
+usedFor: node
+properties:
+  price:
+    type:
+      list: false
+      type: timeseries
+    nullable: false
+    autoIncrement: false
+    name: price
+    description: TODO
+  production:
+    type:
+      list: false
+      type: timeseries
+    nullable: false
+    autoIncrement: false
+    name: production
+    description: TODO
+  shopResult:
+    type:
+      list: false
+      type: direct
+    nullable: true
+    autoIncrement: false
+    name: shopResult
+    description: TODO
```

Comment:
What was the reason for shopResult being here?
ShopPartialBidMatrixCalculationInput takes a list of the PriceProduction object, so this means that ShopResult exists multiple times within ShopPartialBidMatrixCalculationInput.